app/views/application/equality-monitoring/index.njk

```diff
@@ -1,6 +1,6 @@
 {% extends "_content.njk" %}
 {% set applicationPath = "/application/" + applicationId %}
-{% set title = "Equality and diversity" %}
+{% set title = "Your application is now ready to submit. Before you do so, will you complete a 3-minute questionnaire?" %}
```

Comment:
I think “Your application is now complete” can be misinterpreted as “Application submitted”. ie This bit is optional, there’s no opt out, I can close my browser as I’m done.